sdk/security/keyvault/azkeys/autorest.md

```diff
@@ -1,252 +0,0 @@
-## Go
```

Comment:
Deleting this had to feel pretty good...

Reply:
I miss her lol